src/injected/selector-map-helper.ts

```diff
@@ -105,6 +116,14 @@ export class SelectorMapHelper {
                     cardSelectionStoreData,
                 );
                 break;
+            case VisualizationType.AutomatedChecks:
```

Comment:
I'm not sure if this is the right place for this. The function name refers to Adhoc visualizations (and so does the `isAdHocVisualization` method) above. Either we should rename these methods appropriately or we should move this to the assessment method below (`getFilteredSelectorMap`)

Alternatively, we could also move these to the configurations as well if we so choose, like the `getTestViewContainer` but that might be a future refactor.